pkg/pathutils/pathutils.go

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
Nice package, but we will need some tests for this.

Reply:
will add shortly